src/order/models.py

```diff
@@ -75,7 +75,9 @@ def get_shippable_orders(self, delivery_date=None):
         return self.get_queryset().filter(
             delivery_date=delivery_date,
             status=ORDER_STATUS_ORDERED,
-            client__status=Client.ACTIVE)
+            client__status=Client.ACTIVE).order_by(
```

Comment:
Not sure if the order should be defined here. It seems that the ``get_shippable_orders`` method must return the orders ready to be delivered for a given date. That's all. IMO the ``order_by`` statement should be used elsewhere (in your view).